doc/configure

```diff
@@ -134,7 +134,7 @@ def man_to_html(man_name, man_out, builds):
 def convert_man_dir_to_man(
         src_dir, dst_dir, rule, html_mans, builds, gen_html, extra_inputs=None):
     for man in (src_dir).iterdir():
-        man_name = str(man.name).split(".", 1)[0]
```

Comment:
This was required as the logic `str(man.name).split(".", 1)[0]` was returning -

`litani-outcome-table` INSTEAD OF `litani-outcome-table.json`
AND
`litani-run` INSTEAD OF `litani-run.json`
```

Reply:
Is this not the desired behavior? It seems like `man.stem` does the same thing as `str(man.name).split(".", 1)[0]`

Reply:
We need `litani-outcome-table.json` and `litani-run.json` as output but `str(man.name).split(".", 1)[0]` returns `litani-outcome-table` and `litani-run` because of which the schema for both in documentation was breaking earlier.

Reply:
How is what `man.stem` returns any different is my question

Reply:
For a file named `litani-outcome-table.json.scdoc` -

- [x] `man.stem` returns `litani-outcome-table.json`

- [ ] `str(man.name).split(".", 1)[0]` returns `litani-outcome-table`

For a file named `litani-run.json.scdoc` -

- [x] `man.stem` returns `litani-run.json`

- [ ] `str(man.name).split(".", 1)[0]` returns `litani-run`